### ISSUE TITLE
Failed to load resource: 

### ISSUE THREAD

**Issue by isathish** [jlattimer/CRMRESTBuilder#48]
Failed to load resource: the server responded with a status of 404 (Not Found)
XrmSvcToolkit.js:22 Uncaught ReferenceError: Xrm is not defined
```
    at XrmSvcToolkit.js:22
    at XrmSvcToolkit.js:22
    at XrmSvcToolkit.js:22
(anonymous) @ XrmSvcToolkit.js:22
(anonymous) @ XrmSvcToolkit.js:22
(anonymous) @ XrmSvcToolkit.js:22
```
Xrm.RESTBuilder.js:389 Uncaught TypeError: Cannot read property 'context' of undefined
```
    at Object.Xrm.RESTBuilder.GetCrmVersion (Xrm.RESTBuilder.js:389)
    at HTMLDocument.<anonymous> (Xrm.RESTBuilder.js:56)
    at c (jquery1.9.1.min.js:3)
    at Object.fireWith [as resolveWith] (jquery1.9.1.min.js:3)
    at Function.ready (jquery1.9.1.min.js:3)
    at HTMLDocument.H (jquery1.9.1.min.js:3)
Xrm.RESTBuilder.GetCrmVersion @ Xrm.RESTBuilder.js:389
```
```
(anonymous) @ Xrm.RESTBuilder.js:56
c @ jquery1.9.1.min.js:3
fireWith @ jquery1.9.1.min.js:3
ready @ jquery1.9.1.min.js:3
H @ jquery1.9.1.min.js:3
```
favicon.ico:1 Failed to load resource: the server responded with a status of 404 (Not Found)

**Comment by jlattimer** [comment]
This is a problem on your side someplace

**Comment by luis888** [comment]
True, sorry, Thank you for response, the question was silly, « the more you trust the more you find your bugs without asking » , trust will come.

I love your work, I am building  a CRM simple web site over SQL Server (currently old Delphi) with your JSON plugin and the new FOR JSON SQL option, connection works well but I cannot find the use of Query to transmit parameters to be used by the database server program, I see my parameters in the exported React studio code let in a variable for the spreadsheet but nothing arrives in the GET request on my node server.

I am an old experienced developper (I started with Fortran on a TTY in 1976), I had a software editor small firm and I am doing that for fun ?:)?!! and a good friend, Your Product remind me of Delphi in the 90’s, it was a fantastic IDE with  main advantage to have a very useful integrated help with description and always samples.

I am using Visual studio code for debugging the react javascript code if needed.

Best regards

**Comment by luis888** [comment]
Hi, do not bother I found the response to my question in the code, I thought you were using Ajax, sorry again, (trust is better but not yet enough..)

Best Regards



 —————————————————————————————————————————-


True, sorry, Thank you for response, the question was silly, « the more you trust the more you find your bugs without asking » , trust will come.

I love your work, I am building  a CRM simple web site over SQL Server (currently old Delphi) with your JSON plugin and the new FOR JSON SQL option, connection works well but I cannot find the use of Query to transmit parameters to be used by the database server program, I see my parameters in the exported React studio code let in a variable for the spreadsheet but nothing arrives in the GET request on my node server.

I am an old experienced developper (I started with Fortran on a TTY in 1976), I had a software editor small firm and I am doing that for fun ?:)?!! and a good friend, Your Product remind me of Delphi in the 90’s, it was a fantastic IDE with  main advantage to have a very useful integrated help with description and always samples.

I am using Visual studio code for debugging the react javascript code if needed.

Best regards
